### ISSUE TITLE
Nested Ordered Lists

### ISSUE THREAD

**Issue by Sliffcak** [bpampuch/pdfmake#2582]
Given the following dd

```
// playground requires you to assign document definition to a variable called dd

var dd = {
  "content": [
    {
      "ol": [
        {
          "text": "ordere list one"
        },
        {
          "ol": [
            {
              "text": "nested"
            },
            {
              "text": "nested again"
            },
            {
              "ol": [
                {
                  "text": "double nested"
                }
              ]
            }
          ]
        },
        {
          "text": "ordered list two"
        }
      ]
    }
  ]
}
```

Produces the following numbering

![image](https://user-images.githubusercontent.com/19200789/236269967-c0f13f51-b5be-4342-8883-e428bb63a1eb.png)

Is this expected? Is my dd incorrect in some way?

**Comment by Sliffcak** [comment]
I still have yet to figure this out. I am unable to find any solution. Any thoughts?

**Comment by Sliffcak** [comment]
any updates on this?

**Comment by fe-cj** [comment]
You can wrap a list item and the subsequent nested list in an array:

```
var dd = {
  "content": [
    {
      "ol": [
        [
            {
              "text": "ordere list one"
            },
            {
              "ol": [
                {
                  "text": "nested"
                },
                [
                    {
                      "text": "nested again"
                    },
                    {
                      "ol": [
                        {
                          "text": "double nested"
                        }
                      ]
                    }
                ]
              ]
            }
        ],
        {
          "text": "ordered list two"
        }
      ]
    }
  ]
}
```
![image](https://github.com/user-attachments/assets/6bd8fdaf-3c87-4602-97fd-de62a9bf5daa)

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2760.
Released in version 0.2.11.
